x-pack/plugins/lens/public/visualizations/metric/visualization.tsx

```diff
@@ -669,4 +669,76 @@ export const getMetricVisualization = ({
     };
     return suggestion;
   },
+
+  getVisualizationInfo(state: MetricVisualizationState) {
+    const accessors = {
+      metric:
+        state.layerType === 'metricTrendline'
```

Comment:
I don't think that's how the metric vis state works - `state.layerType` will always be `DATA` (https://github.com/elastic/kibana/blob/c0869571b34291e91f2f96baebfe5343906b726f/x-pack/plugins/lens/public/visualizations/metric/visualization.tsx#L381). The trendline accessors are from the secondary hidden layer, they should not be returned from this function.

Reply:
I thought that just another type of layer which user can configure by UI

Reply:
Check out the shape of the state, it doesn't hold an array of layers, there's just one object with properties for the trendline stuff and the regular stuff next to each other.

Reply:
got it, done